### DISCUSSION TITLE
Why I don't have any other option?

### DISCUSSION THREAD

**Discussion by SulimnKbti** [tylernguyen/x1c6-hackintosh#161] [Q&A] [answered]
Hi his is my first time trying to do a Hackintosh with a laptop
and I don't have any option except Reset NVRAM

This is all the files in the USB,
https://drive.google.com/drive/folders/19_uWXWQh-mTO2bDxX-0dsLkeMG_8ubLB?usp=sharing

And I hope someone has a solution
![IMG_8894](https://user-images.githubusercontent.com/97847408/149802124-b202a481-8f39-49a1-9e02-12ed40f3688e.jpg)

**Comment by vanche93** [comment, marked as answer]
Set `ScanPolicy` to `0`
https://github.com/tylernguyen/x1c6-hackintosh/blob/main/EFI/OC/config.plist#L1608

**Reply by SulimnKbti** [reply]
That worked thanks

**Reply by ws1088** [reply]
i got the same issue. I wonder if the instruction should mention that? or is it just me and @SulimnKbti who have the problem?
